scripts/apps/ingest/services/SendService.ts

```diff
@@ -216,7 +216,7 @@ export function SendService(
             return api.save('move', {}, {task: data, allPackageItems: config.sendAllPackageItems}, item)
                 .then((_item) => {
                     $rootScope.$broadcast('item:update', {item: _item});
-                    if (item.lock_user) {
+                    if (item?.lock_user && getAuthoringWorkspace().getItem()?._id === item?._id) {
```

Comment:
Don't use optional chaining where it's not needed. Is is possible that `item` can be `null`? I see optional chaining wasn't used before so I'd guess that not. In that case you also don't need it in ` item?._id`.

Also, don't use type casting in `if` conditions. Do convert it to boolean explicitly - `if (item.lock_user != null)`